tests/integration/src/test/scala/mesosphere/marathon/integration/ExternalVolumesIntegrationTest.scala

```diff
@@ -0,0 +1,81 @@
+package mesosphere.marathon
+package integration
+
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.setup._
+import play.api.libs.json.Json
+
+class ExternalVolumesIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
+
+  override val marathonArgs: Map[String, String] = Map(
+    "enable_features" -> "external_volumes"
+  )
+
+  "ExternalVolume names with options" should {
+    "be considered valid" in {
+      Given("a new app")
+      val app = Json.parse(appWithExtVol).as[raml.App]
+
+      When("The app is deployed")
+      val result = marathon.createAppV2(app)
+
+      Then("The app is created")
+      result should be(Created)
```

Comment:
I don't think we need to test more than just creation of the app here – what will happen with the values serialized into the volume name is up to the provider, which is out of our reach.

Reply:
+1

Reply:
Can we just write an AppsResourceTest ?

Reply:
✅